code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -566,29 +586,29 @@ nobiliumsuppression = INFINITY
 	var/temperature = air.temperature
 	var/pressure = air.return_pressure()
 	var/old_heat_capacity = air.heat_capacity()
-	var/reaction_efficency = min(1/((pressure/(0.1*ONE_ATMOSPHERE))*(max(cached_gases[/datum/gas/plasma][MOLES]/cached_gases[/datum/gas/nitrous_oxide][MOLES],1))),cached_gases[/datum/gas/nitrous_oxide][MOLES],cached_gases[/datum/gas/plasma][MOLES]/2)
-	var/energy_released = 2*reaction_efficency*FIRE_CARBON_ENERGY_RELEASED
-	if ((cached_gases[/datum/gas/nitrous_oxide][MOLES] - reaction_efficency < 0 )|| (cached_gases[/datum/gas/plasma][MOLES] - (2*reaction_efficency) < 0) || energy_released <= 0) //Shouldn't produce gas from nothing.
+	var/reaction_efficency = min(1 / ((pressure / (0.1 * ONE_ATMOSPHERE)) * (max(cached_gases[/datum/gas/plasma][MOLES] / cached_gases[/datum/gas/nitrous_oxide][MOLES], 1))), cached_gases[/datum/gas/nitrous_oxide][MOLES], cached_gases[/datum/gas/plasma][MOLES] * 0.5)
+	var/energy_released = 2 * reaction_efficency * FIRE_CARBON_ENERGY_RELEASED
+	if ((cached_gases[/datum/gas/nitrous_oxide][MOLES] - reaction_efficency < 0 )|| (cached_gases[/datum/gas/plasma][MOLES] - (2 * reaction_efficency) < 0) || energy_released <= 0) //Shouldn't produce gas from nothing.
 		return NO_REACTION
-	ASSERT_GAS(/datum/gas/bz,air)
-	cached_gases[/datum/gas/bz][MOLES] += reaction_efficency
+	ASSERT_GAS(/datum/gas/bz, air)
+	cached_gases[/datum/gas/bz][MOLES] += reaction_efficency * 2.5
 	if(reaction_efficency == cached_gases[/datum/gas/nitrous_oxide][MOLES])
-		ASSERT_GAS(/datum/gas/oxygen,air)
-		cached_gases[/datum/gas/bz][MOLES] -= min(pressure,1)
-		cached_gases[/datum/gas/oxygen][MOLES] += min(pressure,1)
+		ASSERT_GAS(/datum/gas/oxygen, air)
+		cached_gases[/datum/gas/bz][MOLES] -= min(pressure,0.5)
+		cached_gases[/datum/gas/oxygen][MOLES] += min(pressure,0.5)
```

Comment:
esplain

Reply:
overall i'm trying to make BZ a bit easier to make by raising the output and lowering the removal of it due to the reaction_efficency == moles of n2o
I saw many people complaining that BZ is been very slow to make due to this, so i hope that those changes will help people produce more BZ that will be mostly essential to the making of new gases

Reply:
Keep an eye on the state of things, we'll see how it goes